### PR TITLE
Implement selection aggregates

### PR DESCRIPTION
**Pull request by JoomFX**
closes https://github.com/telerik/kendo-themes-private/issues/56

Rendering:

```
<div class="k-selection-aggregates k-grid-selection-aggregates">
    <div class="k-selection-aggregates-item">
        <span class="k-selection-aggregates-item-text">Sum: </span>
        <span class="k-selection-aggregates-item-value">9.00</span>
    </div>
    <div class="k-selection-aggregates-item">
        <span class="k-selection-aggregates-item-text">Avg: </span>
        <span class="k-selection-aggregates-item-value">3.00</span>
    </div>
    <div class="k-selection-aggregates-item">
        <span class="k-selection-aggregates-item-text">Min: </span>
        <span class="k-selection-aggregates-item-value">2.00</span>
    </div>
    <div class="k-selection-aggregates-item">
        <span class="k-selection-aggregates-item-text">Max: </span>
        <span class="k-selection-aggregates-item-value">4.00</span>
    </div>
    <div class="k-selection-aggregates-item">
        <span class="k-selection-aggregates-item-text">Count: </span>
        <span class="k-selection-aggregates-item-value">4</span>
    </div>
</div>
```

### CODE REVIEW

File: packages/bootstrap/scss/grid/_variables.scss [hunk 1]
```diff
@@ -191,6 +199,28 @@ $kendo-grid-selected-text: $kendo-table-selected-text !default;
...
+$kendo-grid-selection-aggregates-text: $kendo-grid-text !default;
+/// Border color of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border: $kendo-grid-border !default;
+/// Border width of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border-width: $kendo-grid-border-width !default;
+/// Spacing between the selection aggregates items
+/// @group grid
+$kendo-grid-selection-aggregates-spacing: k-map-get( $kendo-spacing, 2 );
```
Comment (TeyaVes) [inline packages/bootstrap/scss/grid/_variables.scss:216]: add !default here and below

File: packages/classic/scss/grid/_variables.scss [hunk 2]
```diff
@@ -198,6 +206,28 @@ $kendo-grid-selected-text: $kendo-table-selected-text !default;
...
+$kendo-grid-selection-aggregates-text: $kendo-grid-text !default;
+/// Border color of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border: $kendo-grid-border !default;
+/// Border width of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border-width: $kendo-grid-border-width !default;
+/// Spacing between the selection aggregates items
+/// @group grid
+$kendo-grid-selection-aggregates-spacing: k-map-get( $kendo-spacing, 2 );
```
Comment (TeyaVes) [inline packages/classic/scss/grid/_variables.scss:223]: add !default

File: packages/default/scss/grid/_variables.scss [hunk 3]
```diff
@@ -191,6 +199,28 @@ $kendo-grid-selected-text: $kendo-table-selected-text !default;
...
+$kendo-grid-selection-aggregates-text: $kendo-grid-text !default;
+/// Border color of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border: $kendo-grid-border !default;
+/// Border width of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border-width: $kendo-grid-border-width !default;
+/// Spacing between the selection aggregates items
+/// @group grid
+$kendo-grid-selection-aggregates-spacing: k-map-get( $kendo-spacing, 2 );
```
Comment (TeyaVes) [inline packages/default/scss/grid/_variables.scss:216]: add !default here and below

File: packages/fluent/scss/grid/_variables.scss [hunk 4]
```diff
@@ -240,6 +248,28 @@ $kendo-grid-footer-text: $kendo-table-footer-text !default;
...
+$kendo-grid-selection-aggregates-text: $kendo-grid-text !default;
+/// Border color of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border: $kendo-grid-border !default;
+/// Border width of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border-width: $kendo-grid-border-width !default;
+/// Spacing between the selection aggregates items
+/// @group grid
+$kendo-grid-selection-aggregates-spacing: map.get( $kendo-spacing, 2 );
```
Comment (TeyaVes) [inline packages/fluent/scss/grid/_variables.scss:265]: add !default here and below

File: packages/html/src/grid/tests/grid-selection-aggregates.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en" class="k-no-animations">
+<head>
+    <title>Grid</title>
```
Comment (TeyaVes) [inline packages/html/src/grid/tests/grid-selection-aggregates.html:4]: change title

File: packages/material/scss/grid/_variables.scss [hunk 6]
```diff
@@ -191,6 +199,28 @@ $kendo-grid-selected-text: $kendo-table-selected-text !default;
...
+$kendo-grid-selection-aggregates-text: $kendo-grid-text !default;
+/// Border color of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border: $kendo-grid-border !default;
+/// Border width of the selection aggregates container
+/// @group grid
+$kendo-grid-selection-aggregates-border-width: $kendo-grid-border-width !default;
+/// Spacing between the selection aggregates items
+/// @group grid
+$kendo-grid-selection-aggregates-spacing: k-map-get( $kendo-spacing, 2 );
```
Comment (TeyaVes) [inline packages/material/scss/grid/_variables.scss:216]: add !default here and below

**Review by Juveniel — APPROVED** [review]

**Pull request merged**